tap_trello/streams.py

```diff
@@ -76,59 +76,46 @@ def _get_window_state(self):
         window_state = {}
```

Comment:
^^^
The class needs `config`, `state`, `stream_id` etc (re:pylint), they can be defined as None, or empty dicts by default.